pagesections/ComponentInteraction.js

```diff
@@ -15,18 +15,30 @@ const ComponentInteraction = ( Base ) => class extends Base {
 		};
 	}
 
+	/**
+	 * @param {WebdriverIO.Element} element
+	 * @param {string|number} value
+	 */
 	setValueOnLookupElement( element, value ) {
 		element.$( 'input' ).setValue( value );
 		$( 'body' ).$( this.constructor.OOUI_SELECTORS.LOOKUP_OPTION_WIDGET ).waitForDisplayed();
```

Comment:
Any way to have these `OOUI_SELECTORS` detected by the IDE?

Reply:
Giving the classes a name seems to be enough to help the IDE.